pkg/cephfs/volumemounter.go

```diff
@@ -116,10 +124,36 @@ func mountFuse(mountPoint string, cr *credentials, volOptions *volumeOptions, vo
 		return err
 	}
 
-	if !bytes.Contains(stderr, []byte("starting fuse")) {
+	// Parse the output:
```

Comment:
can you post a sample output here to help understand the following parsing?

Reply:
Oops, this commit was not supposed to be in this PR! (That's what happens when I work late in the evening, apparently :D)  Sorry!

I'm closing this one, will open a new one.